### PR TITLE
Bring back banelings as minions

### PR DESCRIPTION
**Pull request by BraveMole**
## About The Pull Request

Banelings are back. No more pod, it is a minion, if it dies, it dies.
## Why It's Good For The Game
They are cute. Probably a bit too strong for now, I will nerf if needed

## Changelog
:cl:
add: Bring back banelings as minions
/:cl:

### CODE REVIEW

**Comment by 00ze-cyclone** [comment]
This solve nothing to the issue of baneling.
It’s still endlessly reswpaning fucker that can just dive in the middle of the ungaball with no risks at all or way to prevent the gassing.
It already was a minion in effect.

**Comment by TiviPlus** [comment]
Uhh no 90% of the issue was the pod meaning you cant kill them for good

**Comment by coldironwarrior** [comment]
he lives?

**Comment by novaepee** [comment]
so back

**Comment by SimplySmiley123** [comment]
> Uhh no 90% of the issue was the pod meaning you cant kill them for good

And making it a minion that will get spammed out  and doesn't cost anyone anything solves it in what way?

**Comment by mkvrs** [comment]
we love infinite simplemob victories in this house

**Comment by Yttriums** [comment]
🙏🙏🙏🙏🙏

File: code/datums/keybinding/xeno.dm [hunk 1]
```diff
@@ -211,6 +211,26 @@
...
+	description = "Choose a reagent that will be spread upon death. Costs plasma to change."
+	keybind_signal = COMSIG_XENOABILITY_BANELING_CHOOSE_REAGENT
+	hotkey_keys = list("C")
+
+/datum/keybinding/xeno/long_range_sight
+	name = "long_range_sight"
+	full_name = "Boiler: Long Range Sight"
+	description = "Toggles the zoom in."
+	keybind_signal = COMSIG_XENOABILITY_LONG_RANGE_SIGHT
+	hotkey_keys = list("E")
```
Comment (TiviPlus) [inline code/datums/keybinding/xeno.dm:233]: Erm?

File: code/modules/mob/living/carbon/xenomorph/castes/baneling/abilities_baneling.dm [hunk 2]
```diff
@@ -0,0 +1,110 @@
+// ***************************************
+// *********** Baneling Explode
+// ***************************************
+/datum/action/ability/xeno_action/baneling_explode
+	name = "Baneling Explode"
+	action_icon_state = "baneling_explode"
+	desc = "Explode and spread dangerous toxins to hinder or kill your foes. You die."
+	var/static/list/baneling_smoke_list = list(
```
Comment (TiviPlus) [inline code/modules/mob/living/carbon/xenomorph/castes/baneling/abilities_baneling.dm:8]: New vars after old and autodoc

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Rest lgtm

**Comment by Arturlang** [comment]
Could remove acid and neuro, so it can't do much actual damage on it's own

**Comment by efzapa** [comment]
this is just bane but people get to infinitely spam it (now with no cooldown cause minions are SSD mobs) 

the state of TGMC is something these days

**Comment by CVSthePharmacy** [comment]
don't let this get merged, this goes against the entire reason it was removed

**Pull request merged**